<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstWindow</class>
 <widget class="QDialog" name="FirstWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_mascir_video">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttons_horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_video">
      <property name="text">
       <string>Video</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_image">
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttons_horizontalLayout_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
